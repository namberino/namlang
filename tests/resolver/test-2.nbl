var test = "test";
{
    var test = test; // error
}
