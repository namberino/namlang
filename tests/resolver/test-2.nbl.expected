Executing file: tests/resolver/test-2.nbl
On line: 3, Error at 'test': Can't read local variable in its initializer
